dev/kyuubi-extension-spark_3.1/src/test/scala/org/apache/spark/sql/KyuubiExtensionSuite.scala

```diff
@@ -39,6 +39,7 @@ class KyuubiExtensionSuite extends QueryTest with SQLTestUtils with AdaptiveSpar
         "org.apache.kyuubi.sql.KyuubiSparkSQLExtension")
       .config(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key, "true")
       .config("spark.hadoop.hive.exec.dynamic.partition.mode", "nonstrict")
+      .config("spark.hadoop.hive.metastore.client.capability.check", "false")
```

Comment:
this config is for compatible with different Hive, see [HIVE-24608](https://issues.apache.org/jira/browse/HIVE-24608)